docs/index.rst

```diff
@@ -20,21 +17,18 @@ For users who want to start using the toolbox and framework, take a peek at our
    :depth: 2
    :local:
 
-
-
 The Tools
 ---------
 
 The tools are accessible from both a command-line interface (CLI) and a Python API. The CLI automates many operations commonly needed in NWP workflows. The API supports all CLI operations, and additionally provides access to in-memory objects to facilitate more novel use cases. We hope these options will let you integrate the package into your pre-existing bash and Python scripts, and give you some handy tools to use in your day-to-day work with running NWP systems.
 
-
 Configuration Management
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
 | **CLI**: ``uw config -h``
 | **API**: ``import uwtools.api.config``
 
-The config tool suite helps you compare, transform, modify, and even validate your configuration. The package supports YAML, shell, Fortran namelist, and INI file formats. Configuration in any of these formats may express values using Jinja2 syntax. These values can reference others, or compute new values by evaluating mathematical expressions, building paths, manipulating strings, etc.
+The config tool suite helps you compare, transform, modify, and even validate your configuration. The package supports YAML, shell, Fortran namelist, and INI file formats. Configuration in any of these formats may express values using :jinja2:`Jinja2 syntax<templates>`. These values can reference others, or compute new values by evaluating mathematical expressions, building paths, manipulating strings, etc.
```

Comment:
This is an interesting syntax that took a minute to catch what was going on. I appreciate this flexibility and level of DRY-ness.